File: data_structures/basic_array/basic_array.c
/**
 * basic_array.c
 * author: Susan Lunn
 */

#include <stdio.h>

int main(){
    int example_array [5];

    //first we need to fill the array with 0s
    for(int i = 0; i < 5; i++){
        example_array[i] = 0;
    }

    printf("[");
    for(int i = 0; i < 5; i++){
        example_array[i] = i + 1; // can't do i++, because that will return i and then increment that value
        printf("%d", example_array[i]);
        if(i < 4){
            printf(", ");
        }
    }
    printf("]\n");

}

File: data_structures/basic_array/basic_array.java
/**
 * oh hello java! I have missed you!
 * @author Susan Lunn
 * Just a basic array
 */


class BasicArray{

    public static void main(String args[]){
        int[] array = new int[6];
        System.out.print("[");
        for(int i = 0; i < 6; i++){
            array[i] = i + 1;
            if(i < 5)
                System.out.print(array[i] + ", ");
            else
                System.out.print(array[i]);
        }
        System.out.println("]");
    }

}


